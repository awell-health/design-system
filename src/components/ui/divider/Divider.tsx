export const Divider = (): JSX.Element => <div className='h-[1.5px] bg-gray-200 w-full' />;
