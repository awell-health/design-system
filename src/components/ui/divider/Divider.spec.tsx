import * as React from 'react';
import { render } from '@testing-library/react';
import { expect, it, describe } from 'vitest';
import { Divider } from './Divider';

describe('Divider', () => {
  it('match snapshot', () => {
    const { container } = render(<Divider />);
    expect(container).toMatchSnapshot();
  });
});
