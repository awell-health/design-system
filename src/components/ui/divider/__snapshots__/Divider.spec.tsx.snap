// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Divider > match snapshot 1`] = `
<div>
  <div
    class="h-[1.5px] bg-gray-200 w-full"
  />
</div>
`;
